src/lib/constants/constants.js

```diff
@@ -9,7 +9,9 @@ let serviceTemplateFolder = '../../service-templates';
 let sampleDataFolder = '../../sample-data';
 
 module.exports.BASE_TEMPLATE_FILES_LOCATION = path.join(__dirname, '../../templates');
-module.exports.TEMPLATE_FILES_LOCATION = path.join(__dirname, serviceTemplateFolder);
+// TODO: Changing both template file locations to be '../../templates'
+//  Remove functionality to support the src/service-templates in the future
+module.exports.TEMPLATE_FILES_LOCATION = path.join(__dirname, '../../templates');
```

Comment:
nice, this is exactly the hack I had in mind when i first looked at this, glad to see it works